enverus_direct_access/__init__.py

```diff
@@ -41,7 +41,7 @@ def _chunks(iterable, n):
 
 
 class BaseAPI(object):
```

Comment:
In my point of view we should add it to BaseAPI:

    def get_access_token(self):
        raise NotImplementedError